Client/app/src/main/java/at/aau/anti_mon/client/activities/ActivityGameField.java

```diff
@@ -186,10 +188,31 @@ public void onDiceNumberReceivedEvent(DiceNumberReceivedEvent event) {
         moveFigure(location, diceNumber, figure);
     }
 
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onBalanceChangeReceivedEvent(ChangeBalanceEvent event) {
+        int new_balance = event.getBalance();
+        String username = event.getUsername();
+        Log.d("Update_balance", String.valueOf(new_balance));
+        userAdapter.updateUserMoney(username, new_balance);
+    }
+
     private void moveFigure(int location, int diceNumber, ImageView figure) {
         for (int i = 1; i <= diceNumber; i++) {
             if (location == MAX_FIELD_COUNT) {
                 location = 0;
+                // increase here the balance on bank account on 100 Euro
```

Comment:
you forgot to check if it is the same user, now everybody will get the money, instead of only the user that passed the start field